src/cow-react/api/gnosisProtocol/api.ts

```diff
@@ -10,18 +10,18 @@ import {
 } from 'utils/signatures'
 import { APP_DATA_HASH, GAS_FEE_ENDPOINTS, RAW_CODE_LINK } from 'constants/index'
 import { getProviderErrorMessage, registerOnWindow } from 'utils/misc'
-import { environmentName, isBarn, isDev, isLocal, isPr } from '../../utils/environments'
+import { environmentName, isBarn, isDev, isLocal, isPr } from '../../../custom/utils/environments'
```

Comment:
I know, just moving the files. Anyway, replace with full path since we are at it

Reply:
will address in last PR